lib/pharos/kube/client.rb

```diff
@@ -10,7 +10,7 @@ class Client < ::Kubeclient::Client
       def self.for_host(host, version)
         config = Pharos::Kube.host_config(host)
         path_prefix = version == 'v1' ? 'api' : 'apis'
-        api_version, api_group = version.split('/').reverse
+        api_group, api_version = version.split('/')
```

Comment:
I think it was intentional, I was confused by this line too. If version does not have '/', `api_group` would be nil. With this change api_version will be nil.

Reply:
Odd, this does behave differently, but it did still work when testing... needs some specs.

Reply:
Finally found a case where this breaks... seems like these usually only end up getting interpolated into the path as `".../#{api_group}/#{api_version}"`, and in that case it doesn't matter which is which... except the `create_entity` tries to detect the `@api_group` from the given URL, and then automatically set `hash[:apiVersion] = @api_group + @api_version`:
```
no implicit conversion of nil into String
/usr/local/bundle/gems/kubeclient-3.0.0/lib/kubeclient/common.rb:324:in `+'
/usr/local/bundle/gems/kubeclient-3.0.0/lib/kubeclient/common.rb:324:in `create_entity'
/usr/local/bundle/gems/kubeclient-3.0.0/lib/kubeclient/common.rb:185:in `block (2 levels) in define_entity_methods'
/app/lib/pharos/phases/configure_weave.rb:33:in `rescue in ensure_passwd'
/app/lib/pharos/phases/configure_weave.rb:20:in `ensure_passwd'
/app/lib/pharos/phases/configure_weave.rb:16:in `call'
/app/lib/pharos/phase_manager.rb:80:in `block in apply'
/app/lib/pharos/phase_manager.rb:47:in `block in run_serial'
/app/lib/pharos/phase_manager.rb:46:in `map'
/app/lib/pharos/phase_manager.rb:46:in `run_serial'
/app/lib/pharos/phase_manager.rb:58:in `run'
/app/lib/pharos/phase_manager.rb:77:in `apply'
/app/lib/pharos/cluster_manager.rb:88:in `apply_phase'
/app/lib/pharos/cluster_manager.rb:71:in `apply_phases'
/app/lib/pharos/up_command.rb:104:in `configure'
/app/lib/pharos/up_command.rb:47:in `block in execute'
/app/lib/pharos/up_command.rb:45:in `chdir'
/app/lib/pharos/up_command.rb:45:in `execute'
/usr/local/bundle/gems/clamp-1.2.1/lib/clamp/command.rb:63:in `run'
/usr/local/bundle/gems/clamp-1.2.1/lib/clamp/subcommand/execution.rb:11:in `execute'
/usr/local/bundle/gems/clamp-1.2.1/lib/clamp/command.rb:63:in `run'
/usr/local/bundle/gems/clamp-1.2.1/lib/clamp/command.rb:132:in `run'
/app/lib/pharos/root_command.rb:14:in `run'
bin/pharos-cluster:12:in `<main>'
```